src/app/containers/SocialEmbed/common/translations.js

```diff
@@ -1,54 +1,60 @@
 import pathOr from 'ramda/src/pathOr';
 
-const createTranslations = ({ translations, index }) => ({
-  fallback: {
-    text: pathOr(
-      'Content is not available',
-      ['socialEmbed', 'fallback', 'text'],
-      translations,
-    ),
-    linkText: pathOr(
-      'View content on %provider_name%',
-      ['socialEmbed', 'fallback', 'linkText'],
-      translations,
-    ),
-    linkTextSuffixVisuallyHidden: pathOr(
-      ', external',
-      ['socialEmbed', 'fallback', 'linkTextSuffixVisuallyHidden'],
-      translations,
-    ),
-    warningText: pathOr(
-      null,
-      ['socialEmbed', 'fallback', 'warningText'],
-      translations,
-    ),
-  },
+const createTranslations = ({ translations, index }) => {
+  const skipLinkText = pathOr(
+    'Skip %provider_name% content',
+    ['socialEmbed', 'skipLink', 'text'],
+    translations,
+  ).concat(index === undefined ? '' : `, ${index}`);
```

Comment:
I think this reads better - better to have the left hand side of the ternary have the code that is doing something IMO

```suggestion
  ).concat(index > 0 ? `, ${index}` : '');
```